modules/nodes-scheduler-taints-tolerations-adding-machineset.adoc

```diff
@@ -91,5 +168,89 @@ Wait for the machines to be removed.
 $ oc scale --replicas=2 machineset <machineset> -n openshift-machine-api
 ----
 +
+Or:
++
+[source,terminal]
+----
+$ oc edit machineset <machineset> -n openshift-machine-api
+----
++
+[TIP]
+====
+You can alternatively apply the following YAML to scale the machine set:
```

Comment:
I'm suggesting we remove this one because it is essentially identical to the preceding example.